security/backends/elasticsearch/models.py

```diff
@@ -145,6 +151,9 @@ class InputRequestLog(RequestLog):
     class Index:
         name = '{}-input-request-log'.format(settings.ELASTICSEARCH_DATABASE.get('prefix', 'security'))
 
+    def __str__(self):
+        return '{} {}'.format(_('input request log'), self.short_id)
```

Comment:
:( there should be gettext and not gettext lazy.. It is method.

Reply:
I refactored that into mixin, now it's valid lazy gettext